docs/how-tos/using-sentry-locally.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Using Sentry.io for Local Development
+---
+
+FxA uses [sentry.io](https://sentry.io) to monitor errors and performance in production / staging. FxA developers can also use [sentry.io](https://sentry.io) during local development. Here are a few scenarios where enabling sentry locally might be helpful:
+
+- Validating that an error reproduced locally aligns with an error reported in staging or production.
+- Auditing performance / errors after exploratory testing, functional testing, or a load test.
+- Gaining a better understanding of calls made by a certain workflow or endpoint. 
+
+To enable sentry for local development, simply set these environment variables to their respective DSN settings. When these variables are defined, sentry will be activated. 
+
+SENTRY_ENV=local
+[SENTRY_DSN_ADMIN_PANEL](https://sentry.io/settings/mozilla/projects/fxa-admin-panel/keys/)
+[SENTRY_DSN_AUTH](https://sentry.io/settings/mozilla/projects/fxa-auth/keys/)
+[SENTRY_DSN_CONTENT](https://sentry.io/settings/mozilla/projects/fxa-content/)
+[SENTRY_DSN_CUSTOMS](https://sentry.io/settings/mozilla/projects/fxa-customs/)
+[SENTRY_DSN_EVENT_BROKER](https://sentry.io/settings/mozilla/projects/fxa-event-broker/keys/)
+[SENTRY_DSN_GRAPHQL_API](https://sentry.io/settings/mozilla/projects/fxa-graphql-api/keys/)
+[SENTRY_DSN_PAYMENTS](https://sentry.io/settings/mozilla/projects/fxa-payments/keys/)
+[SENTRY_DSN_PROFILE](https://sentry.io/settings/mozilla/projects/fxa-profile/keys/)
+[SENTRY_DSN_SUPPPORT_PANEL](https://sentry.io/settings/mozilla/projects/fxa-support-panel/)
+[SENTRY_DSN_BROWSERID](https://sentry.io/settings/mozilla/projects/fxa-browserid-verify/keys/)
+
+_Each variable name links to the sentry.io page containing the DSN value. Asking a fellow engineer for the settings might be the fastest route though..._ 😉
```

Comment:
You can use the `:::tip` syntax here to make it fancier.  [Example](https://github.com/mozilla/ecosystem-platform/pull/193/files#diff-2eccd1ae7e1999de667f30e5211759efc7ef51bd76e44f15b7a3b565f7643a13R4)